app/oryx-app-mllib/pom.xml

```diff
@@ -26,7 +26,7 @@
   <artifactId>oryx-app-mllib</artifactId>
   <packaging>jar</packaging>
   <name>Apps: Spark MLlib</name>
-  <description>Speed and batch components of machine learning applications built on
+  <description>Batch components of machine learning applications built on
```

Comment:
I don't think this is accurate since it does intend to hold speed and batch components in general